lib/location/method_gnss.c

```diff
@@ -689,6 +692,30 @@ static void method_gnss_pvt_work_fn(struct k_work *item)
 			location_core_event_cb_error();
 		}
 	}
+
+	if ((pvt_data.flags & NRF_MODEM_GNSS_PVT_FLAG_NOT_ENOUGH_WINDOW_TIME) &&
```

Comment:
Maybe a comment on top of the if-statement on what is done here just like there is in the previous if-statement.